### DISCUSSION TITLE
How to use ` --ignore-path` with two files?

### DISCUSSION THREAD

**Discussion by babakfp** [eslint/eslint#17870] [Q&A] [answered]
https://eslint.org/docs/latest/use/configure/ignore#using-an-alternate-file

There wasn't anything in the docs. I tried multiple ways but didn't work.

I'm looking for something like this

```json
{
    "lint": "eslint --ignore-path=[.gitignore,.eslintignore] ."
}
```

**Comment by kecrily** [comment, marked as answer]
We have `--ignore-pattern` flag. When you can't find relevant information, try `eslint --help`. This is also mentioned in the documentation

> Paths are relative to the current working directory. This is also true of paths passed in via the --ignore-pattern [command](https://eslint.org/docs/latest/use/command-line-interface#--ignore-pattern).

```shell
eslint --ignore-pattern=.gitignore,.eslintignore .
```

This should work

**Reply by babakfp** [reply]
It worked. Thanks.

**Reply by platinumazure** [reply]
One thing worth noting: `--ignore-path` is for specifying a gitignore-like *file containing glob patterns* to be ignored, whereas `--ignore-pattern` allows glob patterns (which ESLint should ignore) to be specified on the command line.

So if your goal is to make sure ESLint doesn't lint .gitignore and .eslintignore, using `--ignore-pattern` is correct. But if your goal is to have ESLint use ignore patterns from both .gitignore and .eslintignore, I don't believe that is supported.

**Reply by babakfp** [reply]
Hey

Thanks for that.

I was looking to combine the content inside both `.gitignore` and `.eslintignore` and use that to ignore patterns.

I'm using eslint v`8.56.0` and it does what I was looking for, so, maybe it's a feature that is not yet documented?

**Reply by wouterkroes** [reply]
Indeed, it is not implemented (yet) -> https://github.com/eslint/eslint/issues/9794
